modules/flowable-engine-common/src/test/java/org/flowable/common/engine/impl/el/JsonNodeELResolverTest.java

```diff
@@ -0,0 +1,85 @@
+package org.flowable.common.engine.impl.el;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
The import order is defined in https://github.com/flowable/flowable-engine/tree/master/ide-settings:

- statics
- java
- javax
- org
- com
- liquibase
- everything else